routes/api/v1/forecast.js

```diff
@@ -0,0 +1,62 @@
+var express = require('express');
+var router = express.Router();
+
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+const fetch = require('node-fetch');
+const yaml = require("js-yaml");
+const fs = require("fs");
+
+const User = require('../../../lib/models/users');
+const Forecast = require('../../../lib/pojos/forecast');
+
+
+router.get('/', (request, response) => {
+  var api_key = request.body.api_key;
+  var location = request.query.location;
+
+  if (User.find(api_key) === api_key) {
+    fetchCoords = async () => {
+      const key = await yaml.safeLoad(fs.readFileSync("./config.yml", "utf8"))["google_geocoding_api"];
+
+      try {
+        let response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${location}&key=${key}`);
+        let geocodingData = await response.json();
+        let latLong = await geocodingData["results"][0]["geometry"]["location"];
+
+        return latLong;
+      } catch (err) {
+        console.log(err)
+      }
+    }
+
+    fetchForecastData = async () => {
+      const key = await yaml.safeLoad(fs.readFileSync("./config.yml", "utf8"))["dark_sky_api"];
```

Comment:
woah! this is COOL

Reply:
Yes, the package is called `js-yaml`, similar to `application.yml` in a rails app